pyproject.toml

```diff
@@ -73,7 +73,7 @@ Twitter = "https://twitter.com/LitestarAPI"
 annotated-types = ["annotated-types"]
 attrs = ["attrs"]
 brotli = ["brotli"]
-cli = ["jsbeautifier", "uvicorn[standard]", "uvloop>=0.18.0"]
+cli = ["jsbeautifier", "uvicorn[standard]", "uvloop>=0.18.0; sys_platform != 'win32'"]
```

Comment:
Why are we even specifying `uvloop`? Is there a reason that it needs to be `>=0.18.0`? If not, we can just allow `uvicorn` to decide whether to install `uvloop` or not since it's part of the `standard` extras of `uvicorn` as seen [here](https://github.com/encode/uvicorn/blob/40b99b8436c0c261e3a85d10e291424072946292/pyproject.toml#L36-L45).

Reply:
It is for 3.12 compatibility. Uvicorn doesn't have the same constraint, hence the spec.

Reply:
Found the commit for you: https://github.com/litestar-org/litestar/pull/2396/commits/34bc1957d39155a30d54de2b65d3ebfe21953630